lib/default.nix

```diff
@@ -239,6 +239,15 @@ in {
     inherit stdenv lib haskellLib srcOnly;
   };
 
+  # Do coverage of a project
```

Comment:
This is coverage of a *package*, right? The other one is a project.

Reply:
Addressed https://github.com/input-output-hk/haskell.nix/pull/762/commits/326fcbfc733756b9d2c2f1ac39503f6c74365104.